plugins/inputs/redfish/redfish.go

```diff
@@ -147,6 +163,14 @@ func (r *Redfish) Init() error {
 		return fmt.Errorf("did not provide the computer system ID of the resource")
 	}
 
+	if r.IncludeTagSets == nil {
+		r.IncludeTagSets = []string{tagSetChassisLocation}
+	}
```

Comment:
You can't do it this way as otherwise the user has no chance to deselect both options. Please set the initial `IncludeTagSets` in  `init()` (lowercase) to allow the user to explicitly set this empty.

Reply:
Ohh I see